clients/da-vinci-client/src/main/java/com/linkedin/davinci/stats/KafkaConsumerServiceStats.java

```diff
@@ -155,6 +157,10 @@ public void recordDelegateSubscribeLatency(double value) {
     delegateSubscribeLatencySensor.record(value);
   }
 
+  public void recordWaitAfterUnsubscribeLatency(int value) {
+    waitAfterUnsubscribeLatencySensor.record(value);
```

Comment:
I think we're leaking more casts because we're doing an int parameter here on this interface.  Why not cast to int in here and remove the casts from other parts of the code?

Reply:
actually, why cast at all? record takes a long.

Reply:
Reverted back to using `long` everywhere. The `record()` function seems to only take `double` values, so I'm just going to keep it as `long` everywhere instead of casting to `int` at the end?